src/common/stats/StatsManager.h

```diff
@@ -88,9 +128,19 @@ class StatsManager final {
                                   double pct);
     static void readAllValue(folly::dynamic& vals);
 
-private:
+
+    void prometheusSerialize(std::ostream& out) /*const*/ override;
+
     static StatsManager& get();
 
+private:
+    static constexpr bool isStatIndex(int32_t i);
+    static constexpr bool isHistoIndex(int32_t i);
+    static constexpr std::size_t physicalStatIndex(int32_t i);
+    static constexpr std::size_t physicalHistoIndex(int32_t i);
```

Comment:
Inline these short functions

Reply:
Inline is just hint for compiler, it will inline in release mode.

Reply:
I means why not put these **one line** function in their declaration location?  It will be readable for others.

Reply:
Ok.